content/table-schema/contents.lr

```diff
@@ -190,7 +190,7 @@ This lexical formatting may be modified using these additional properties:
   within the number. The default value is ".".
 * **groupChar**: A string whose value is used to group digits within the
   number. The default value is null. A common value is "," e.g. "100,000".
```

Comment:
@pwalsh i've also just spotted that we aren't removing percentage here - isn't that part of this commit? The percentage example would then either disappear or move to the stripNonNumeric section.